sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -225,6 +225,32 @@ Table {counter: tableNumber}. Terminology.
 
 |===
 
+==== Event Terminology  [[event-terminology]]
+
+:events-spec: https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:interface.event
+
+For the purposes of clarity when talking about events in this specification we
+will split events into two categories:
+
+- *Graph-limited events*: These are events returned from a queue submission
+which is recorded to a `command_graph`. These events are only valid for defining
+dependencies for other nodes inside a `command_graph`. These events cannot be
+waited on or used as dependencies for normal SYCL operations. They also cannot
```

Comment:
`normal SYCL operation` is used twice in the spec, while `eager SYCL operation` is used 6 times in the spec.  Shall we use one term for it?